gcloud/logging/handlers/transports/background_thread.py

```diff
@@ -127,7 +127,14 @@ def enqueue(self, record, message):
             self._entries_condition.acquire()
             if self.stopping:
                 return
+            record_dict = record.__dict__
+            for blacklisted in ("args", "msg", "message"):
```

Comment:
UPDATE: I just realized @waprin used double quotes when originally checking this code in.

Sigh.